### ISSUE TITLE
Customizing the AimMode property

### ISSUE THREAD

**Issue by omateusteles** [afriscic/BarcodeScanning.Native.Maui#80]
I believe it would be interesting to use the `AimMode` property without the red dot in the middle of the screen, allowing those who use the library to customize it the way they prefer and still use the functionality that `AimMode` provides.

**Comment by afriscic** [comment]
Hello.
Yes, ti would be nice to have, it's just that it is quite a bit of work, especially to test everything...

And I'm not entirely sure what is the best way to do it, I think there are two approaches:

1. Do everything native, enable different shapes, sizes and colours. There would be a lot of code to support everything natively and it wouldn't look the same on all platforms.
2. Implement `IContentView` to have the ability to add MAUI child controls to `CameraView`. I think it would be easier to implement but I'm afraid that it would be very had to test and potentially have layout problems

**Comment by omateusteles** [comment]
Yes, it would definitely be a lot of work, so I don't believe you should develop this but leave the responsibility to whoever wants to customize the scanner. What I suggested here as an improvement would just be this possibility of hiding the red dot in the middle of the screen so that I could make my own template in `AimMode`
